Formula/vdirsyncer.rb

```diff
@@ -1,8 +1,9 @@
+# coding: utf-8
```

Comment:
Minor belated notation, this shouldn't be necessary any more. We've logic rammed into the core to treat formulae files as UTF-8.